src/main/java/eisenwave/elytra/listeners/SneakListener.java

```diff
@@ -32,30 +32,29 @@ public void onToggleSneak(final PlayerToggleSneakEvent event) {
                     return player.getEquipment() != null
                             && player.getEquipment().getChestplate() != null
                             && player.getEquipment().getChestplate().getType() == Material.ELYTRA;
-                }
-        ).then(() -> {
-            Player player = event.getPlayer();
+                }).then(() -> {
+                    Player player = event.getPlayer();
 
-            // Grounded
-            if (player.getLocation().getY() - (double) player.getLocation().getBlockY() < 0.0001) {
-                if (event.isSneaking()) {
-                    playerManager.getPlayer(player).setChargeUpTicks(0);
-                }
-                else {
-                    if (playerManager.getPlayer(player).getChargeUpTicks() >= this.plugin.config().chargeupTicks) {
-                        if (!this.plugin.getLaunchCooldownManager().canUse(player.getUniqueId())) {
-                            sendCooldownMessage(player);
-                            return;
+                    // Grounded
+                    if (player.getLocation().getY() - (double) player.getLocation().getBlockY() < 0.0001) {
+                        if (event.isSneaking()) {
+                            playerManager.getPlayer(player).setChargeUpTicks(0);
+                        } else {
+                            if (playerManager.getPlayer(player).getChargeUpTicks() >= playerManager.getPlayer(player)
```

Comment:
Again, this should probably be the main change to this file. I'm seeing a lot of other format-related changes.